src/reboost/build_glm.py

```diff
@@ -9,6 +9,8 @@
 from lgdo.lh5 import LH5Iterator, LH5Store
 from numpy.typing import ArrayLike
 
+from reboost import utils
+
 log = logging.getLogger(__name__)
```

Comment:
I would guess this is the culprit for the duplicate messages, maybe it is not good to init the logger before setting up colorlog?

Reply:
The comment was meant to go to cli.py, but ayway this was wrong.